tick/stateful/functions.go

```diff
@@ -83,6 +84,9 @@ func init() {
 	statelessFuncs["day"] = &day{}
 	statelessFuncs["month"] = &month{}
 	statelessFuncs["year"] = &year{}
+
+	// Humanize functions
+	statelessFuncs["size"] = &size{}
```

Comment:
Should the function name be something like `humanBytes` or something along those lines?